app/src/main/java/fr/tabbya/instantvisio/VideoCallActivity.java

```diff
@@ -18,6 +27,30 @@ protected void onCreate(Bundle savedInstanceState) {
         enableJavascript();
         String visioUrl = getVisioUrl();
         Log.d("[VideoCallActivity]", "starting video call on address: " + visioUrl);
+
+        mWebView.getSettings().setJavaScriptEnabled(true);
+        mWebView.getSettings().setJavaScriptCanOpenWindowsAutomatically(true);
+
+        mWebView.getSettings().setSaveFormData(true);
+        mWebView.getSettings().setSupportZoom(false);
+        mWebView.getSettings().setCacheMode(WebSettings.LOAD_NO_CACHE);
```

Comment:
the code is more readable when encapsulating several lines doing a specific action under a function with an explanatory name (Clean code): you could avoid the reader developer to read all these lines by extracting them in a function, like

```java
setWebviewSettings() { ... }
```